### ISSUE TITLE
Include "CN" column number symbol

### ISSUE THREAD

**Issue by ogarcia** [adobe-fonts/source-code-pro#291]
If you use [vim](https://www.vim.org/) with [airline](https://github.com/vim-airline/vim-airline) and powerline enabled you need a "CN" symbol at U+E0A3 like the "LN" symbol.

Sample: https://github.com/ryanoasis/powerline-extra-symbols/blob/master/img/fontforge.png
Code: https://github.com/vim-airline/vim-airline/blob/master/doc/airline.txt#L397

Could this be added to Source Code Pro?

**Comment by fourpastmidnight** [comment]
You don't "need" it. But, it _is_ nice to have. Having said that, there is an option, at least with `vim-airline` to set the font face to be used in the powerline. So, your regular editor font could be SoureCode Pro while the powerline font could be, for example, FiraCode.

Check out the documentation on `vim-airline`.

**Comment by ogarcia** [comment]
> [..] So, your regular editor font could be SoureCode Pro while the powerline font could be, for example, FiraCode.

Yes, I know. But the idea is to use Source Code Pro for everything and in that case it is interesting to have the CN symbol. :wink:

**Comment by fourpastmidnight** [comment]
Did you check out the Nerd Font website? They take fonts and add the Powerline symbols to them. I prefer Microsoft's Cascadia Code, for example, and Nerd Font patched the font with the Powerline symbols and called it Kaskadia Cove.

I beleieve they've done the same with SourceCode Pro. Other fonts they've patched are FiraCode and Hasklig, among many others.

**Comment by pauldhunt** [comment]
@ogarcia I’ll add this in the next revision.
